Tests/Twig/Extension/MediaExtensionTest.php

```diff
@@ -101,7 +101,9 @@ public function getProvider()
     public function getTemplate()
     {
         if (is_null($this->template)) {
-            $this->template = $this->createMock('Twig_TemplateInterface');
+            $this->template = $this->createMock(
+                class_exists('Twig_Template') ? 'Twig_Template' : 'Twig_TemplateInterface'
```

Comment:
The `NEXT_MAJOR` comment disappeared.

Reply:
I tried `class_exists` as @core23 suggested but it [fails](https://travis-ci.org/sonata-project/SonataMediaBundle/builds/281986768) on PHP < 5.6. Not sure if my fault or twig specific. I go back to previous test with `version_compare`.